tests/test_common.js

```diff
@@ -251,50 +251,125 @@ test('restoreOverriddenRequests can be called more than once', t => {
   t.end()
 })
 
-test('stringifyRequest', function(t) {
-  const exampleOptions = {
-    method: 'POST',
-    port: 81,
-    proto: 'http',
-    hostname: 'example.test',
-    path: '/path/1',
-    headers: { cookie: 'fiz=baz' },
-  }
+test('stringifyRequest', async t => {
```

Comment:
At @gr2m's suggestion I've been trying to keep the tap subtest trees flat as much as possible. Do you mind updating these?

Also if the functions under test aren't async, I think it's clearer to make the test functions synchronous and use `t.end()`.

Reply:
These have been flattened